src/components/input/index.tsx

```diff
@@ -17,7 +17,7 @@ type SharedProps = {
   onFocus?: FocusEventHandler<HTMLInputElement>;
   autoFocus?: boolean;
   name: string;
-  type?: 'text' | 'number';
+  type?: 'text' | 'number' | 'password';
```

Comment:
Could we add `password` in stories type -> options

Reply:
Thanks, added